src/teleop/gui/src/components/GimbalControls.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+    <div class="wrap">
+        <div v-show='false' id="key">
+          <input v-on:keydown="keymonitordown">
+          <input v-on:keyup="keymonitorup">
+        </div>
+        <p>Gimbal Controls</p>
+    </div>
+</template>
+
+<script>
+
+import ROSLIB from "roslib"
+
+export default {
+
+    data() {
+    return {
+
+        keyboard_pub: null,
+
+        inputData : {
+            w_key: 0,
+            a_key: 0,
+            s_key: 0,
+            d_key: 0
+        } 
+    }
+    },
+
+    created: function () {
+        document.addEventListener('keydown', this.keymonitordown);
+        document.addEventListener('keyup', this.keymonitorup);
+
+        this.keyboard_pub = new ROSLIB.Topic({
+            ros : this.$ros,
+            name : '/gimbal_control',
+            messageType : 'mrover/GimbalCmd'
+        });
+    },
+
+  methods: {
+    keymonitordown: function(event) {
```

Comment:
Can the method names be snake case or camel case (whatever the rest of the GUI code uses)?

Reply:
Seems like default Vue method names are camel case so that may be best